packages/ui/package.json

```diff
@@ -51,12 +51,17 @@
     "lint:style:fix": "yarn lint:style --fix"
   },
   "dependencies": {
+    "@dnd-kit/core": "^6.1.0",
     "@kaoto-next/uniforms-patternfly": "^0.7.14",
+    "@kaoto/xml-schema-ts": "workspace:*",
     "@kie-tools-core/editor": "0.32.0",
     "@kie-tools-core/notifications": "0.32.0",
     "@types/uuid": "^10.0.0",
+    "@types/xml-name-validator": "^4.0.3",
+    "@visx/shape": "^3.12.0",
     "ajv": "^8.12.0",
     "ajv-formats": "^3.0.0",
+    "chevrotain": "10.5.0",
```

Comment:
Is it expected to use 10.5.0 which was released 2 years ago? There is a 11.0.3 which was released a yea rago